### ISSUE TITLE
I can't make database transactions  

### ISSUE THREAD

**Issue by shahendaeldeeb** [fragmenta/fragmenta-cms#38]
Functions exposed in file named database_psql.go doesn't allow me to send transactions to database as it doesn't have a specific function that deal with pq driver transaction function.
I think we can solve this issue by generating new function that uses pq driver transaction functions.

Thanks in advance

**Comment by kennygrant** [comment]
I agree it'd be nice to support transactions explicitly somehow. You can access the driver in the meantime I think.

**Comment by shahendaeldeeb** [comment]
first thanks for your reply ,
I think transactions is very common and essential  functionality, so it would be nice if it's handled in next release/update.
